src/libraries/Microsoft.PowerFx.Connectors/Tabular/CdpTableDescriptor.cs

```diff
@@ -1,16 +1,21 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+using System.Collections.Generic;
+using Microsoft.PowerFx.Types;
+
 namespace Microsoft.PowerFx.Connectors
 {
-    internal readonly struct CdpTableDescriptor
+    public readonly struct CdpTableDescriptor
```

Comment:
>struct [](http://example.com/codeflow?start=20&length=6)

why struct?

Reply:
no idea, not important, I'll make it a class